README.md

```diff
@@ -1,3 +1,19 @@
+Compiled test available can be downloaded from this link [drive](https://drive.google.com/open?id=0BwB2Eafip96vbjU4RWduT1BUdnc)
+
+you will have to modify emulator/common/Makefile.include to provide the correct path for the binaries after extracting 
+
+Alternatively following link contains the compiler [drive](https://drive.google.com/open?id=0BwB2Eafip96vYzkwc0Z0clFpdVU) 
+
+To use the compiler extract it to suitable folder
+
+```bash 
+#take a back up of /opt/riscv if it already exists
+sudo rm /opt/riscv 
+sudo ln -s <folder> /opt/riscv
```

Comment:
This seems strange and unnecessary. Can't they just set ```$RISCV``` to point to the extracted folder?